### ISSUE TITLE
[AsyncApi] Accept the generation without "schemas:"

### ISSUE THREAD

**Issue by adrien1212** [sngular/scs-multiapi-plugin#238]
# Problem
According to the example available [on asyncapi documentation](https://www.asyncapi.com/docs/tutorials/create-asyncapi-document)

```yaml
asyncapi: '2.6.0'
info:
  title: Streetlights API
  version: '1.0.0'
  description: |
    The Smartylighting Streetlights API allows you
    to remotely manage the city lights.
  license:
    name: Apache 2.0
    url: 'https://www.apache.org/licenses/LICENSE-2.0'
servers:
  mosquitto:
    url: mqtt://test.mosquitto.org
    protocol: mqtt
channels:
  light/measured:
    publish:
      summary: Inform about environmental lighting conditions for a particular streetlight.
      operationId: onLightMeasured
      message:
        name: OnLightMeasured
        payload:
          type: object
          properties:
            id:
              type: integer
              description: Id of the streetlight.
            lumens:
              type: integer
              description: Light intensity measured in lumens.
            sentAt:
              type: string
              format: date-time
              description: Date and time when the message was sent.
```

## Actual
The class `OnLightMeasured` is not generated (because it isn't in the `schemas:` section)

## Expected
A class `OnLightMeasured` with attributs, getter and setter

**Comment by 5uso-sng** [comment]
Hi, thanks for the report! We are preparing a new release for the plugin addressing this issue, which will hopefully be ready soon

**Comment by 5uso-sng** [comment]
The release should be ready to go, fix version is 4.8.1
